go.mod

```diff
@@ -238,6 +238,8 @@ replace github.com/hashicorp/go-immutable-radix => github.com/hashicorp/go-immut
 
 replace github.com/hashicorp/go-hclog => github.com/hashicorp/go-hclog v0.12.2
 
+replace k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20221207184640-f3cff1453715
```

Comment:
I don't understand how this `replace` directive can have any effect when it's the same revision as on line 225. I think the directive should be removed, or do you mean to upgrade to a newer revision?